kubernetes_test.go

```diff
@@ -411,6 +411,22 @@ func TestKubernetesClusters_GetKubeConfig(t *testing.T) {
 	require.Equal(t, blob, got.KubeconfigYAML)
 }
 
+func TestKubernetesClusters_GetKubeConfigWithExpiry(t *testing.T) {
+	setup()
+	defer teardown()
+
+	kubeSvc := client.Kubernetes
+	want := "some YAML"
+	blob := []byte(want)
+	mux.HandleFunc("/v2/kubernetes/clusters/deadbeef-dead-4aa5-beef-deadbeef347d/kubeconfig", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, http.MethodGet)
```

Comment:
It might be nice to check that the query parameter is set as expected.
See `TestKubernetesClusters_GetCredentials_WithExpirySeconds` for a hint.

Reply:
Thank you, I've added it